site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Toloka'
+linkTitle: 'Toloka'
+weight: 2
+---
+
+To have your dataset annotated through Toloka, simply establish
+a project in CVAT set the pricing, and let Toloka annotators
+take care of the annotations for you.
+
+See:
+
+- [Glossary](#glossary)
+- [Preconditions](#preconditions)
+- [Creating Toloka project](#creating-toloka-project)
+- [Adding tasks and jobs to the Toloka project](#adding-tasks-and-jobs-to-the-toloka-project)
+- [Adding instructions for annotators to the Toloka project](#adding-instructions-for-annotators-to-the-toloka-project)
+- [Toloka pool setup](#toloka-pool-setup)
+- [Changing Toloka pool](#changing-toloka-pool)
+- [Reviewing annotated jobs](#reviewing-annotated-jobs)
+  - [Accepting job](#accepting-job)
+  - [Rejecting job](#rejecting-job)
+- [Moving Toloka project to archive](#moving-toloka-project-to-archive)
+
+## Glossary
+
+This page contains several terms used to describe interactions between systems and actors.
+Refer to the table below for clarity on how we define and use them.
+
+<!--lint disable maximum-line-length-->
+
+| Term      | Explanation                                                                                                                                                                                                                                                                   |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Toloka    | Toloka, developed by Yandex, is a crowdsourcing platform that allows users to assign tasks to a broad group of participants, often termed "crowd workers". In the context of this article, when we mention Toloka, we are specifically referring to one of its UI interfaces. |
+| CVAT      | CVAT, developed by OpenCV, is a visualization tool designed for annotating video and image data for computer vision tasks. In this article's context, when we reference CVAT, we mean one of its UI interfaces.                                                               |
```

Comment:
And CVAT is not developed by OpenCV :) 
I believe we may omit developers information here